og_ui/og_ui.module

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * @file
+ * Organic Groups UI.
+ */
+
+use Drupal\Core\Entity\EntityTypeInterface;
+
+/**
+ * Implements hook_entity_bundle_field_info_alter().
+ */
+function og_ui_entity_bundle_field_info_alter(&$fields, EntityTypeInterface $entity_type, $bundle) {
+  if (isset($fields['og_group'])) {
+    $fields['og_group']->setDisplayOptions('view', [
+      'type' => 'og_ui_group_subscribe',
+      'weight' => 0,
```

Comment:
is `weight` and `label` needed here? (didn't check but assumed they would be the default)